transform/kubernetes/kubernetes.go

```diff
@@ -268,6 +269,10 @@ func (k *KubernetesTransformPlugin) getWhiteOuts(obj unstructured.Unstructured)
 		return true
 	}
 
+	if groupKind.Group == extensionsGroup {
+		return true
+	}
```

Comment:
For future self. I initially balked at this idea of totally excluding everything from the `extensions` group, but looking at the [package information](https://pkg.go.dev/k8s.io/kubernetes@v1.24.3/pkg/apis/extensions/v1beta1#pkg-variables) it's pretty clear that nothing lives here anymore except conversions to other kube resource versions.

My suspicion (link would be nice) is that there was a time kube was okay with have extensions in k/k, but they decided separate controllers were better and therefor having an `extensions` group was no longer appropriate.